src/components/entity/CredentialItem.vue

```diff
@@ -22,20 +22,20 @@
             <v-icon small class="fake-link">{{ mdiOpenInNew }}</v-icon>
           </a>
         </div>
-        <div v-if="effectiveDate">
-          <span>Effective:&nbsp;</span>
-          <span>{{ effectiveDate | formatDate }}</span>
-        </div>
         <div v-if="reason">
           <span>Reason:&nbsp;</span>
           <span>{{ reason }}</span>
         </div>
-        <div>
+        <div v-if="!expired">
           <v-icon small>{{ mdiShieldCheckOutline }}</v-icon>
-          <span class="fake-link vertical-align-middle"
-            >Credential verified</span
+          <router-link :to="`${baseRoute}/credential/${credId}`"
```

Comment:
Can also pass an object with the route name and the url parameters

Reply:
change completed